algviz/graphs/graphs.py

```diff
@@ -0,0 +1,36 @@
+import pygraphviz as pgv
+
+
+def nodesToLocations(nodes, edges, algo="dot"):
+    """ Maps node to their coordinates.
+    """
+    G = pgv.AGraph()
+    G.graph_attr['fixedsize'] = True
+    for node in nodes:
+        G.add_node(node, fixedsize=True, width=node.width, height=node.height)
```

Comment:
We also do not always want our nodes to be a fixed size, in general. There will be times when we want nodes in a graph to be differently sized.